### ISSUE TITLE
Suggestion: Please provide a way to remove one or more character's data

### ISSUE THREAD

**Issue by IBHalliwell** [giulianoriccio/Lua#45]
Hello,

A friend of mine logged into his characters using my workstation (which is what I use as my primary computer for FFXI).  Tonight when I went to look for the various crafting kits on my characters, I ended up seeing his 16 characters, too.

It would be wonderful if there was a method to remove those 16 characters, without touching my own.

At a minimum, though, a method (or instructions) to do a complete removal / reset of all character data, too.

Thank you.

**Comment by giulianoriccio** [comment]
Hello,
I'm sorry but I'm not playing FFXI anymore and this project fork is not maintained since a few years ago.

If you are still experiencing the issue reported here, please resubmit it on the official [Windower/Lua repository](https://github.com/Windower/Lua).

Thank you for using my addons.

**Comment by IBHalliwell** [comment]
Thanks!  I did just open an issue where you suggest.

BTW, I came here via the web page - http://wiki.windower.net/doku.php?id=addons:findall

It gives this location as where to come get support and might be why you keep getting people dropping by looking for help.

Just suggesting you might want to have someone update the page and not point here, if this is no longer the place to get support.

BTW, one last question:

Would it be possible for Find / Findall (LUA version) to track the gil of a character (both via equipment and in the Mog House)?  If so, how?  Thanks! :)

**Comment by giulianoriccio** [comment]
Thank you, I've changed the url.

I added the gil already when I was working on it, infact it's stored in the
data the addon uses as its own storage, but they must have removed it along
the way or maybe I didn't have the time to find a way to show it in the
results.

Try asking on the official issue tracker, it shouldn't be hard to add.

Il ven 2 giu 2017, 04:46 IBHalliwell <notifications@github.com> ha scritto:
